css/index.css

```diff
@@ -143,4 +158,110 @@ footer nav {
 footer nav a {
     color: white;
     text-decoration: none;
-}
\ No newline at end of file
+}
+ nav{
+     display:flex;
+     flex-flow: row stretch;
+     justify-content: space-between;
+     align-items:baseline;
+     
+ }
```

Comment:
This could be nav a { } instead of a class name if it is the only place on the page where you want to target a nav a element.